Assets/MRTK/Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -78,6 +78,51 @@ public bool IsOpaque
             }
         }
 
+        /// <inheritdoc />
+        public bool ProjectionOverrideEnabled
+        {
+            get
+            {
+                ProjectionOverride projectionOverride;
+                if(TryGetProjectionOverrideComponent(out projectionOverride, false))
+                {
+                    return projectionOverride.EnableOverride;
+                }
+                return false;
+            }
+            set
+            {
+                ProjectionOverride projectionOverride;
+                if (TryGetProjectionOverrideComponent(out projectionOverride, true))
+                {
+                    projectionOverride.EnableOverride = value;
```

Comment:
The only other minor thing I can think of is, even when setting to false, the projection override MonoBehaviour will still exist on the camera itself, so OnPreCull will always get called (but no-op). This is honestly probably fine, though if we wanted to be hyper senstive we could also just delete the component when turning it off.

I don't have a real opinion on this, but wanted to call it out.

Reply:
I went up and back on this when writing the code.  I finally chose not to remove the component when ReadingMode is disabled because my completely speculative intuition on this is that an app will be turning it on and off during the app's lifetime so leaving it reduces the places where things can go wrong after it's initially created.  But I could be convinced otherwise.